Changes

```diff
@@ -1,5 +1,14 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+2.3.0 2022-06-09
+
+ [Features]
+ - Gives more freedom when configuring (#134, #129, #96)
```

Comment:
Usually we have listed the PRs in decreasing order.

Reply:
Oh. Is that better somehow?

Reply:
No, it is just the way it becomes since I start from the newest and go back in time. They are no ordered, and can jump. Nobody has commented it before. I have not thought it to be important.